community/kernel/src/main/java/org/neo4j/kernel/impl/factory/CommunityEditionModule.java

```diff
@@ -99,6 +100,8 @@ public CommunityEditionModule( PlatformModule platformModule )
 
         GraphDatabaseFacade graphDatabaseFacade = platformModule.graphDatabaseFacade;
 
+        dependencies.satisfyDependency( SslPolicyLoader.create( config, logging.getInternalLogProvider() ) ); // for bolt and web server
```

Comment:
The SslPolicyLoader doesn't reject the legacy settings, it just incorporates the old settings into the new framework which then bolt/web uses and it should yield the same result. This already had tests in place.